cmd/generate_gatewayapi_httproute.go

```diff
@@ -23,11 +24,24 @@ func generateGatewayApiHttpRouteCommand() *cobra.Command {
 		Use:   "httproute",
 		Short: "Generate Gateway API HTTPRoute from OpenAPI 3.0.X",
 		Long:  "Generate Gateway API HTTPRoute from OpenAPI 3.0.X",
-		RunE:  runGenerateGatewayApiHttpRoute,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			outputFormat, err := cmd.Flags().GetString("output-format")
+			if err != nil {
+				return err
+			}
+
+			oasPath, err := cmd.Flags().GetString("oas")
```

Comment:
`generateGatewayAPIHTTPRouteOAS` is already filled with that required input param, isn't it?

Reply:
yes, it is - fixed. reverted to `StringVar`/`StringVarP` pattern as it's a bit more terse.